### PR TITLE
Added PNG of favicon to assets

### LINKED ISSUE #43: Add favicon
**Add favicon**
I would like to add a favicon.Please `/assign` this to me as a part of GSSOC'21

### PR DESCRIPTION
**Pull request by cryptofox17**
### Related Issue 

Added the finalized favicon in PNG format to assets folder

Closes: #43 

### Describe the changes you've made

Added the transparent PNG file to assets.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Mention any unusual behaviour of your code (Write NA if not)

NA

## Additional Info (optional)

NA

## Checklist:

- [ ] My code follows the guidelines of this project.
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly whereever it was hard to understand.
- [x] My changes generate no new warnings.
- [ ] Any dependent changes have been merged and published in downstream modules.

### CODE REVIEW

**Review by adarshvulli — APPROVED** [review]

**Comment by avinal** [comment]
Hello @cryptofox17, do you know how to create a favicon out of these files. If yes then do it or let me know so that I could guide you. Or ping me in Discord.

**Comment by cryptofox17** [comment]
@avinal do you mean adding the html code for it? 
Or making generating a ico file and adding that?

**Comment by avinal** [comment]
Yeah, @cryptofox17, I meant generating ico files and adding site metadata to HTML, it won't take much time and will be a much better contribution.

File: favicon_package_PNG/site.webmanifest [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+    "name": "",
```
Comment (avinal) [inline favicon_package_PNG/site.webmanifest]:
```suggestion
    "name": "PixelVibe",
```
Add the name

File: favicon_package_PNG/site.webmanifest [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    "short_name": "",
```
Comment (avinal) [inline favicon_package_PNG/site.webmanifest]:
```suggestion
    "short_name": "PV",
```

**Comment by cryptofox17** [comment]
@avinal done

**Comment by avinal** [comment]
@cryptofox17, I see now that you haven't added it to the base folder, you have instead added it to `favicon_package_PNG` folder. Can you please move them out of the folder? one reason is that when favicons are in some other folder it may not load properly or as fast. Sorry I didn't notice earlier.

**Review by avinal — COMMENTED** [review]
@cryptofox17, I see now that you haven't added it to the base folder, you have instead added it to `favicon_package_PNG` folder. Can you please move them out of the folder? one reason is that when favicons are in some other folder it may not load properly or as fast. Sorry I didn't notice earlier. Move all the files out of the folder except the png image.

**Comment by cryptofox17** [comment]
@avinal please check if the changes are fine now

**Review by avinal — COMMENTED** [review]
Rest look great, but maybe you forgot to delete this folder, move this picture to `assets` `favicon_package_PNG/favicon_pixelVibe.png`

**Comment by cryptofox17** [comment]
@avinal let me know if its fine now

**Review by avinal — APPROVED** [review]
Looks great now.

**Review by adarshvulli — APPROVED** [review]

**Pull request merged**